ethereum_history_api/circuits/lib/src/transaction.nr

```diff
@@ -4,6 +4,8 @@ use dep::std::unsafe::zeroed;
 global MAX_TRANSACTION_TREE_DEPTH = 7;
 global MAX_TRANSACTION_KEY_LEN = 4;
 
+type TxType = u8;
```

Comment:
We have the same one in receipts. Remove there and add an import from here

Reply:
I removed the tx implementation (that I accidentally added)